Revit_oM/Settings/RevitSettings.cs

```diff
@@ -26,31 +26,31 @@
 
 namespace BH.oM.Adapters.Revit.Settings
 {
+    [Description("General settings that are applicable to all actions performed by the instance of Revit adapter.")]
     public class RevitSettings : BHoMObject
     {
         /***************************************************/
         /****             Public Properties             ****/
         /***************************************************/
 
+        [Description("Socket connection settings.")]
         public ConnectionSettings ConnectionSettings { get; set; } = new ConnectionSettings();
+
+        [Description("Revit family load settings. Limited functionality at the moment.")]
```

Comment:
Does not add anything not added by the type description. Would remove.

Reply:
Maybe elaborate on the "Limited functionality" would add some useful info

Reply:
Agreed @alelom, but unfortunately this is a legacy prototype that I have not cracked yet - would not dare to elaborate on its limitations... This is on my roadmap for the next quarter.